EIPS/eip-transaction-bundles.md

```diff
@@ -0,0 +1,229 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 7727
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.